task-launcher/src/tasks/shared/trials/afcStimulus.js

```diff
@@ -242,6 +242,8 @@ function getButtonHtml(task) {
   // TODO: add trial_type column to math item bank
   if (stimulus.trialType === 'instructions') {
     return "<button class='primary'>%choice%</button>";
+  } else if (task === 'vocab'|| task === 'trog'){
+    return "<button class='image-medium'>%choice%</button>"
```

Comment:
We assign the button class in line `385`, we should probably keep all of it there so that when I update the layoutConfig, I can just do all of it in one place.